chpl/chpl-service/src/main/java/gov/healthit/chpl/search/dao/ListingSearchDao.java

```diff
@@ -66,14 +66,14 @@ public List<ListingSearchResult> getListingSearchResults() {
     }
 
     private List<ListingSearchResult> convertToListingSearchResults(List<ListingSearchEntity> entities)
-        throws EntityRetrievalException, DateTimeParseException, NumberFormatException  {
+            throws EntityRetrievalException, DateTimeParseException, NumberFormatException  {
```

Comment:
I know there are a lot of whitespace compatibility things going on, and it's also really hard to read through PRs with all these whitespace updates. Is it possible to undo these? Restore them? Something else?

Reply:
@kekey1  and I have discussed this problem and I have updated some of my setting to fix this going forward.  The "older" code changes will have these whitespace changes, though.